src/Umbraco.Infrastructure/Persistence/NPocoSqlExtensions.cs

```diff
@@ -1183,6 +1344,19 @@ public static Sql<ISqlContext> Delete<TDto>(this Sql<ISqlContext> sql)
             return sql;
         }
 
+        /// <summary>
+        /// Deletes records from a table based on a predicate.
+        /// </summary>
+        /// <typeparam name="TDto">Table definitio.</typeparam>
+        /// <param name="sql">SqlConext</param>
+        /// <param name="predicate">A predicate to transform and append to the Sql statement (WHERE clause).</param>
+        /// <returns></returns>
+        public static Sql<ISqlContext> Delete<TDto>(this Sql<ISqlContext> sql, Expression<Func<TDto, bool>> predicate)
```

Comment:
This doesn't seem to be used. If we don't need it, could we remove, just to avoid building up the code maintenance burden?